.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# How To Contribute
+To keep our repositories as uniform as possible, and work together effectively please consider the following:
+
+## Our Git Flow
+We use two primary branches.  `master` and `development`.  `development` is what you'll create pull requests against, and should be deployed to staging environments for testing. Once enough work has gone in to create a  deliverable piece value, and all testing is complete, we should merge `development` into `master`, which will then be deployed to a production environment.  Ideally `master` should always represent what is in production. 
+
+### Contributing to the project
+Always branch, and use the below naming schema for branches. Once you've completed the work for that branch, push it up to github and create a Pull Request against development.
+
+#### Naming Branches
+We prefix our branch names to make it easier to see what branches relate to.
+
+* `feature/<branch-name>`  - a new feature
+* `fix/<branch-name>` - fixes an issue
+* `style/<branch-name>` - small front end updates
```

Comment:
I'm wondering the usefulness of this. Do we ever look at a list of branches and try to assign a category to them? A branch is personal (and transient for that matter), the owner should know what that is. Also, these long names suck in terminal.